github/ci/prow-deploy/files/jobs/kubevirt/containerdisks/containerdisks-presubmits.yaml

```diff
@@ -1,5 +1,36 @@
 presubmits:
   kubevirt/containerdisks:
+  - always_run: true
+    optional: false
+    annotations:
+      testgrid-create-test-group: "false"
+    cluster: ibm-prow-jobs
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 4h0m0s
+    labels:
+      preset-podman-in-container-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+    max_concurrency: 11
+    name: pull-containerdisks-test
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - make test
+        env:
+        - name: GIMME_GO_VERSION
+          value: "1.19"
+        image: quay.io/kubevirtci/golang:v20230105-1dbefc0
+        name: ""
+        resources:
+          requests:
+            memory: 4Gi
+        securityContext:
```

Comment:
Do we really need this for unit tests?

Reply:
If the podman in container setup is not required - the job doesn't require privileged.

Reply:
I don't think it is, tbh I just copied it from the other jobs. The tests/linting are not running `make cluster-up`.

Reply:
@brianmcarey Can we drop the `docker-mirror-proxy` too?